### PR TITLE
LPS-120376 Create single sql for other team to import

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

File: benchmarks/build.xml [hunk 1]
```diff
@@ -3,7 +3,24 @@
 <project basedir="." default="run" name="benchmarks">
 	<import file="../build-common.xml" />
 
-	<target name="build-sample-sql">
+	<property name="sample-data-sql" value="sample-data-${sample.sql.db.type}.sql" />
```
Comment (tinatian) [inline benchmarks/build.xml:10]: @slnn 
There is no "sample-data-${sample.sql.db.type}.sql" in https://github.com/liferay/liferay-portal/tree/master/benchmarks. It is confusing you have this property setting here.

I guess you need this for our benchmark project, but this is not a good reason to put it here.

Tina.

Comment (tinatian) [reply benchmarks/build.xml:10]: @slnn Sorry, I misunderstood you. This is not for property reading, it is for output, right?

File: benchmarks/build.xml [hunk 2]
```diff
@@ -69,14 +86,51 @@
...
 		<delete includeemptydirs="true">
 			<fileset
 				dir="."
-				includes="*.csv,*.sql,benchmarks-actual.properties,output/**,temp/**"
+				includes="*.csv,*.sql,benchmarks-actual.properties,liferay/**,output/**,temp/**"
 			/>
 		</delete>
 	</target>
+
+	<target depends="clean, build-sample-sql" name="merge-sql">
```
Comment (tinatian) [inline benchmarks/build.xml:113]: @slnn Who is going to use this "merge-sql"?
If I want to run sample sql builder to get a sample sql file to import into my database, what should I do?

**Comment by dantewang** [comment]
Just run this ant merge-sql


Tina Tian <notifications@github.com> 于 2020年10月22日周四 12:25写道：

**Comment by tinatian** [comment]
@dantewang I just had a talk with Lily, the goal is I run a command and got an sql which can be imported into my database, then I start portal with that database, everything works fine.
@slnn just told me that we have resource action related issue when starting up from a sql built from sample sql builder. 

Let's work on that first, then we go back to this.

Tina.

**Comment by slnn** [comment]
Hi @tinatian 

Update for the previous issues I mentioned:
1>For the ResurceAction Issue, we have two ways to fix it
Fix it on SampleSQLBuilder side:https://github.com/dantewang/liferay-portal/pull/1788
@dantewang  want to fix it on portal side:https://github.com/dantewang/liferay-portal/pull/1790

2>For the encoding issue on Windows, double check it with QA and more Windows users, the origin issue caused by the way that QA import data

So I resend it in:
Portal side:https://github.com/dantewang/liferay-portal/pull/1792
Benchmark side:https://github.com/dantewang/liferay-benchmark-ee/pull/432

Lily

**Pull request closed without merging**